### PR TITLE
[#155867191] editable prop is always passed down so just set to whate…

### PR DESCRIPTION
**Pull request by jasoneo016**
…ver the prop is. The issue at hand was that this comparison checks if its true then set it true, if its false, also set it to true.

### CODE REVIEW

**Review by MattFoley — APPROVED** [review]
👍

File: src/MentionsTextInput.js [hunk 1]
```diff
@@ -578,7 +578,7 @@ export default class MentionsTextInput extends Component {
           maxLength={this.props.maxLength ? this.props.maxLength : Number.MAX_SAFE_INTEGER}
           enablesReturnKeyAutomatically={this.props.enablesReturnKeyAutomatically ? this.props.enablesReturnKeyAutomatically : false}
           underlineColorAndroid={this.props.underlineColorAndroid ? this.props.underlineColorAndroid : 'black'}
-          editable={this.props.editable ? this.props.editable : true}
+          editable={this.props.editable}
```
Comment (armandgray) [inline src/MentionsTextInput.js:581]: why are we doing this?

Comment (MattFoley) [reply src/MentionsTextInput.js:581]: Because muted users shouldn't be able to edit the text field, will this cause other problems?

**Review by armandgray — APPROVED** [review]
👍

**Review by armandgray — APPROVED** [review]
👍

File: src/MentionsTextInput.js (hunk 1 above)
Comment (jasoneo016) [reply src/MentionsTextInput.js:581]: It shouldn't. The cases where a user should not be able to edit the text input is being handled in `ChatInputToolbar`. They aren't able to edit it, I tried hitting enter, and that doesn't do anything either. The only problem I kind of see is, you are able to select the text, and select all and copy it

**Review by MattFoley — APPROVED** [review]
👍

**Pull request merged**